recipe/meta.yaml

```diff
@@ -7,11 +7,18 @@ package:
 source:
   url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
   sha256: 4903bf96110bbab8ed2c68533f90565ebb76aa367d9e4df38e51bf727927c125
+  patches:
+    # Use the conda provided cbc binary, not those bundled in tar-ball:
+    - pulp.cfg.linux.patch
+    - pulp.cfg.osx.patch
 
 build:
-  number: 0
+  number: 1
   script:
+    - rm -rf "pulp/solverdir/cbc/"
     - {{ PYTHON }} -m pip install . -vv
+    # Remove bundled cbc binaries, using coin-or-cbc provided cbc instead:
+    - rm "${SP_DIR}/pulp/solverdir/cbc"
```

Comment:
I tried removing these _before_ running pip install, but then the manifest complains they are missing. Thus mimicked the preexisting line removing the pre-compiled Windows DLL from the installed files.